CHANGELOG.md

```
@@ -1,5 +1,12 @@
 # Changelog
 
+## [0.9.12](https://github.com/justsomesites/reframed/compare/v0.9.11...v0.9.12) (2025-09-01)
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)